js/freewheel.js

```diff
@@ -742,7 +742,12 @@ OO.Ads.manager(function(_, $) {
      * @method Freewheel#fw_onSlotStarted
      */
     var fw_onSlotStarted = function() {
-      amc.ui.adVideoElement.removeAttr('controls');
+      // adVideoElement may be null for overlays
+      if (currentPlayingSlot &&
+          currentPlayingSlot.getTimePositionClass() === tv.freewheel.SDK.TIME_POSITION_CLASS_OVERLAY &&
```

Comment:
isn't this supposed to check if it's a linear ad (not overlay) and then remove the attribute?